scripts/shared/clusters.sh

```diff
@@ -82,6 +83,10 @@ function create_kind_cluster() {
     fi
 
     echo "Creating KIND cluster..."
+    if [[ "${cluster_cni[$cluster]}" == "ovn" ]]; then
+        deploy_kind_ovn
+        return
+    fi
```

Comment:
```suggestion
    fi

```

Sorry missed this nit the first time